code/controllers/subsystems/initialization/atlas.dm

```diff
@@ -76,6 +78,19 @@ var/datum/controller/subsystem/atlas/SSatlas
 
 	QDEL_NULL(maploader)
 
+	InitializeSectors()
+
+	var/chosen_sector
+
+	if(config.current_space_sector)
+		chosen_sector = config.current_space_sector
+	else
+		chosen_sector = current_map.default_sector
+
+	var/datum/space_sector/selected_sector = SSatlas.possible_sectors[chosen_sector]
+
+	current_sector = selected_sector
```

Comment:
You need to add error-handling/validation there to handle the case of a invalid sector in the config.current_space_sector.
You should also unit-test the current_map.default sector to ensure that this is a valid sector.